### PR TITLE
window function: function name of error message should be lowercase

### PR DESCRIPTION
**Pull request by tcmichael**
### What problem does this PR solve? 
function name of error message should be lowercase
Closes https://github.com/pingcap/tidb/issues/11009

### What is changed and how it works?
wrap `windowFunc.F` with `strings.ToLower` 

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by tcmichael** [comment]
@SunRunAway PTAL

File: planner/core/logical_plan_builder.go [hunk 1]
```diff
@@ -3148,7 +3148,7 @@ func (b *PlanBuilder) buildWindowFunctions(p LogicalPlan, groupedFuncs map[*ast.
 				return nil, nil, err
 			}
 			if desc == nil {
-				return nil, nil, ErrWrongArguments.GenWithStackByArgs(windowFunc.F)
+				return nil, nil, ErrWrongArguments.GenWithStackByArgs(strings.ToLower(windowFunc.F))
```
Comment (lamxTyler) [inline planner/core/logical_plan_builder.go]: Does `ErrWindowInvalidWindowFuncUse` need to lower the function name?

Comment (tcmichael) [reply planner/core/logical_plan_builder.go]: `ErrWindowInvalidWindowFuncUse` is not mentioned in #11009.
I think it's another issue.

Comment (SunRunAway) [reply planner/core/logical_plan_builder.go]: #11508

**Pull request merged**